presto-native-execution/src/test/java/com/facebook/presto/spark/PrestoSparkNativeQueryRunnerUtils.java

```diff
@@ -145,18 +169,22 @@ protected ImmutableList<Module> getNativeExecutionModules()
         return moduleBuilder.build();
     }
 
-    protected Path getBaseDataPath()
+    private static Path getBaseDataPath()
```

Comment:
Should we make this method `synchronized` since it'll be access by different classes concurrently?

Reply:
E2E tests are probably not running concurrently, but we can add it, good point. Thanks